cluster-cm/src/main/java/com/sequenceiq/cloudbreak/cm/ClouderaManagerSetupService.java

```diff
@@ -364,6 +364,21 @@ public void waitForHosts(Set<InstanceMetaData> hostsInCluster) throws ClusterCli
         clouderaManagerPollingServiceProvider.startPollingCmHostStatus(stack, client, privateIps);
     }
 
+    @Override
+    public void waitForHostsHealthy(Set<InstanceMetaData> hostsInCluster) throws ClusterClientInitException {
+        Cluster cluster = stack.getCluster();
+        String user = cluster.getCloudbreakAmbariUser();
+        String password = cluster.getCloudbreakAmbariPassword();
+        ApiClient client;
+        try {
+            client = clouderaManagerApiClientProvider.getV31Client(stack.getGatewayPort(), user, password, clientConfig);
+        } catch (ClouderaManagerClientInitException e) {
+            throw new ClusterClientInitException(e);
+        }
+        clouderaManagerPollingServiceProvider.startPollingCmHostStatusHealthy(
```

Comment:
is it intended not to use the result returned?

Reply:
This, iirc, came from another similar poller. Doesn't mean it's right though. If it does need to check ... would prefer tracking as a follow-up.

Reply:
This is essentially waiting for success, or a timeout. Ref: waitForHosts in the same class which I believe is used during regular Upscale.